src/constants.js

```diff
@@ -0,0 +1,3 @@
+export const categoryList = ['Grocery', 'Technology', 'Events', 'Appointment'];
+
+export const priorityList = ['Low', 'Moderate', 'High'];
```

Comment:
Nice keeping these in separate constant file.

Reply:
Refactored the code as per the comments  @codecrook